client/js/overlays/players.js

```diff
@@ -120,7 +163,8 @@ onLoad(function() {
   });
   onMessage('rename', function(args) {
     const oldName = playerName;
-    playerName = args;
+    if(args)
+      playerName = args;
```

Comment:
If the name on the server is set to `""`, the client should not simply ignore that and keep the previous name. I don't mind enforcing a non-empty name but that shouldn't be here.